jest.config.js

```diff
@@ -11,6 +11,12 @@ module.exports = {
     '@stencil/core/testing': '<rootDir>/testing/index.js',
     '@utils': '<rootDir>/src/utils',
   },
+  coverageDirectory: './test-results/',
+  coverageReporters: ["json", "lcov", "text", "clover", "json-summary"],
```

Comment:
The first four are jest defaults - json-summary is something we could use in the future to more easily track quality over time.

Reply:
My thought here is that we follow [YAGNI](https://martinfowler.com/bliki/Yagni.html) and wait until we need it
```suggestion
  coverageReporters: ["json", "lcov", "text", "clover"],
```